plugins/modules/df_service.py

```diff
@@ -88,6 +88,10 @@
     type: bool
     required: False
     default: False
+  tags:
```

Comment:
Comment about line 36 above (GH didn't let me comment directly on there):
This module's `crn` parameter refers to the enviornment CRN, rather than the service's one. This is weird and inconsistent with `df_service_info`, where the parameter is called `environmentCrn`. Also, having a `name` alias for this parameter is very confusing, IMO.

Reply:
good spotting. I had to update this module to handle the requirement for both df_crn and env_crn as the original implementation only used the env_crn. This should now be clearer.